flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x64"
```

Comment:
This architecture is invalid.

Why do you want to restrict ? You never answered.